sklearnex/tests/test_run_to_run_stability.py

```diff
@@ -73,23 +74,31 @@ def eval_method(X, y, est, method):
     est.fit(X, y)
 
     if method:
-        if method != "score":
-            res = getattr(est, method)(X)
-        else:
-            res = est.score(X, y)
+        res = call_method(est, method, X, y)
 
     if not isinstance(res, Iterable):
-        res = [res]
+        results = [_as_numpy(res)] if res is not est else []
+    else:
+        results = []
+        for i in res:
+            # Nearest Neighbors radius_neighbors causes this if statement
+            # It is unique in returning a numpy array of numpy arrays in
+            # a list

```

Comment:
if radius neighbors is being skipped would we still need this?

Reply:
Good catch.  Let me know and I will delete it.

Reply:
Based on the comment it seems like yes - if there are problems with CI after removing then can re-add it.

Reply:
Removed.